hedera-node/hedera-network-admin-service-impl/src/main/java/com/hedera/node/app/service/networkadmin/impl/handlers/NetworkGetAccountDetailsHandler.java

```diff
@@ -194,19 +193,13 @@ private static List<TokenRelationship> getTokenRelationships(
             ReadableTokenStore readableTokenStore,
             ReadableTokenRelationStore tokenRelationStore) {
         final var tokenRelationshipList = new ArrayList<TokenRelationship>();
-        var tokenNum = account.headTokenNumber();
+        var tokenId = TokenID.newBuilder().tokenNum(account.headTokenNumber()).build();
```

Comment:
Rather than use default, I've been copying from the AccountID (in Account), which reduces the "surface area" of places that still assume there will never be another shard or realm.

Reply:
Once `Account` has all IDs this will not be an issue. I think it will be done by Kim

Reply:
Yes, Account will have headTokenId by then and just use that.